paasta_tools/cli/schemas/adhoc_schema.json

```diff
@@ -86,7 +86,7 @@
                 "maximum": "1000000",
                 "exclusiveMinimum": false
             },
-            "cpu_burst_pct": {
+            "cpu_burst_add": {
```

Comment:
As soon as this version of paasta-tools gets installed on the git server for yelpsoa-configs, any pushes that modify services that still have cpu_burst_pct will start failing.

I think that's fine - just make sure you have a yelpsoa-configs CR ready to merge that changes all our uses of cpu_burst_pct, and when you ship this change make sure to run puppet manually on the git server, then merge the change.

Reply:
Cool.